Library/Homebrew/test/test_tap.rb

```diff
@@ -155,7 +155,7 @@ def test_git_variant
   end
 
   def test_private_remote
-    skip "HOMEBREW_GITHUB_API_TOKEN is required" unless ENV["HOMEBREW_GITHUB_API_TOKEN"]
+    skip "HOMEBREW_GITHUB_API_TOKEN is required" unless GitHub.api_token
```

Comment:
The goal of this PR is so that you won't need to set `HOMEBREW_GITHUB_API_TOKEN` if you've got HTTPS cloning setup locally. I think `brew tests` fits with that.